app/src/main/res/values/strings.xml

```diff
@@ -372,4 +372,7 @@
     <string name="never_show_again">Never show again</string>
     <string name="show_manager_update_dialog_on_launch">Show update message on launch</string>
     <string name="show_manager_update_dialog_on_launch_description">Shows a popup notification whenever there is a new update available on launch.</string>
+    <string name="less">Less</string>
+    <string name="any_version">Any version</string>
+    <string name="any_package">Any package</string>
```

Comment:
Move and rename these to make things consistent with the other bundle dialog strings.